File: QE-selenium/python/Activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as dirver:
    dirver.get("https://training-support.net/webelements/dynamic-controls")
    print(dirver.title)
    print(dirver.find_element(By.XPATH,"//input[@id='textInput']").is_enabled())
    dirver.find_element(By.XPATH,"//button[@id='textInputButton']").click()
    print(dirver.find_element(By.XPATH,"//input[@id='textInput']").is_enabled())
    dirver.quit()
    

File: QE-selenium/java/Activity7.java
package demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity7 {

    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net/webelements/dynamic-controls");

        System.out.println(driver.getTitle());

        System.out.println(driver.findElement(By.xpath("//input[@id='textInput']")).isEnabled());

        driver.findElement(By.xpath("//button[@id='textInputButton']")).click();

        System.out.println(driver.findElement(By.xpath("//input[@id='textInput']")).isEnabled());

        driver.quit();
    }
}
